extensions/sql-database-projects/src/test/baselines/sqlProjPropertyRead.xml

```diff
@@ -10,7 +10,6 @@
     <Platform>x64</Platform>
     <OutputPath>CustomOutputPath\Dacpacs\</OutputPath>
     <DefaultCollation>Japanese_CI_AS</DefaultCollation>
-    <ModelCollation>1041,CI</ModelCollation>
```

Comment:
nit: not that this is actually used in the tests, but shouldn't the 1033 one be removed? The 1041 one is the one that matches with Japanese?

Reply:
So thorough!  Yeah, you're right.